bigframes/enums.py

```diff
@@ -27,3 +27,5 @@ class DefaultIndexKind(enum.Enum):
     #: ``n - 3``, ``n - 2``, ``n - 1``, where ``n`` is the number of items in
     #: the index.
     SEQUENTIAL_INT64 = enum.auto()
+    # A completely null index incapable of indexing or alignment.
+    EMPTY = enum.auto()
```

Comment:
Let's align this with the exception type, either `NULL` here or `EmptyIndexError` there.

Reply:
renamed to null